operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -2066,6 +2066,10 @@ metaMonitoring:
   serviceMonitor:
     # -- If enabled, ServiceMonitor resources for Prometheus Operator are created
     enabled: false
+    # -- Create standard Mimir alerts in Prometheus Operator via a PrometheusRule CRD
+    mimirAlerts: false
+    # -- Create standard Mimir recording rules in Prometheus Operator via a PrometheusRule CRD
+    mimirRules: true
```

Comment:
needs to go over to prometheusRule as per https://github.com/grafana/mimir/pull/2609#discussion_r1006611988